Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -121,6 +122,17 @@ public static async Task LaunchStoreForReviewAsync()
         /// </summary>
         public PackageVersion FirstVersionInstalled { get; }
 
+        /// <summary>
+        /// Gets the previous version of the app that was installed.
```

Comment:
In the `IsAppUpdated` property and here we should call out that these are only available the very first time the app is run after it is updated.